src/main/java/uk/gov/ons/fwmt/census/smoketest/gateway/JobServiceHelper.java

```diff
@@ -33,24 +36,25 @@ public boolean checkAppIsRunning() {
 
     HttpEntity<String> request = new HttpEntity<String>(headers);
     ResponseEntity<String> response = restTemplate
-        .exchange(url + "/actuator/health", HttpMethod.GET, request, String.class);
+        .exchange(url + "/health", HttpMethod.GET, request, String.class);
     String result = response.getBody();
 
     // true if we can access the Job Service
     return (result != null) && result.contains("\"status\":\"UP\"");
   }
 
-  public boolean canAccessRabbitQ(String qname) {
+  public List<String> canAccessRabbitQ() {
```

Comment:
This method name is no longer accurate, and should probably be `fetchAccessibleRabbitQueues` or similar.